features/overflow-clip.yml

```
@@ -0,0 +1,9 @@
+name: "overflow: clip"
+description: "The `overflow: clip` CSS declaration clips an element's overflowing content to the box that's defined by the `overflow-clip-margin` property."
```

```suggestion
description: "The `overflow: clip` CSS declaration clips an element's overflowing content to the box that's defined by the `overflow-clip-margin` property. Unlike `overflow: hidden`, it allows you to hide overflow on one axis while keeping overflow on the other axis visible."
```

It also disables programmatic scrolling, but I don't think that's as key as the single axis behavior.